src/functionalTest/java/uk/gov/hmcts/reform/prl/controllers/AddCafcassOfficerControllerFunctionalTest.java

```diff
@@ -40,24 +32,40 @@ public class AddCafcassOfficerControllerFunctionalTest {
     @Autowired
     protected ServiceAuthenticationGenerator serviceAuthenticationGenerator;
 
-    @Before
-    public void setUp() {
-        this.mockMvc = webAppContextSetup(webApplicationContext).build();
-    }
+    private static final String C100_SEND_TO_GATEKEEPERJUDGE = "requests/call-back-controller-send-to-gatekeeperForJudge.json";
+
 
-    @MockBean
-    private AddCafcassOfficerService addCafcassOfficerService;
+    private final String targetInstance =
+        StringUtils.defaultIfBlank(
+            System.getenv("TEST_URL"),
+            "http://localhost:4044"
+        );
+
+    private final RequestSpecification request = RestAssured.given().relaxedHTTPSValidation().baseUri(targetInstance);
 
     @Test
     public void givenRequestBody_whenAdd_cafcass_officer_about_to_submit_then200Response() throws Exception {
         String requestBody = ResourceLoader.loadJson(VALID_REQUEST_BODY);
```

Comment:
Are we not creating a fresh case?

Reply:
Tried to create a fresh case but isAuthorised function in the method throwing error due to use of authorisation tokens twice , tried a day to find the root cause of the error , hence using the json input for all the methods using isAuthorise function .... fyi @shashisk97 @yogendraU